fiftyone/operators/delegated.py

```
@@ -605,54 +616,60 @@ def execute_operation(
+                err = ExecutionResult(error=traceback.format_exc())
+                try:
+                    self.set_failed(
+                        doc_id=operation.id,
+                        result=err,
+                        update_pipeline=operation.parent_id,
+                    )
+                except Exception:
+                    logger.exception(
+                        "Failed to mark operation %s as FAILED", operation.id
+                    )
+                return err
```

Pylint (4.0.2)

[warning] Unused variable 'e'

(W0612)